service/frontend/workflow_handler.go

```diff
@@ -855,6 +882,10 @@ func (wh *WorkflowHandler) GetWorkflowExecutionHistoryReverse(ctx context.Contex
 func (wh *WorkflowHandler) PollWorkflowTaskQueue(ctx context.Context, request *workflowservice.PollWorkflowTaskQueueRequest) (_ *workflowservice.PollWorkflowTaskQueueResponse, retError error) {
 	defer log.CapturePanic(wh.logger, &retError)
 
+	if wh.isStopped() {
```

Comment:
this was missing for PollWorkflowTaskQueue and PollActivityTaskQueue.. any idea why? pretty much all other methods have it. without this, the shut down calls get retried uselessly